es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 
 <br>
 
 **40. Step 4: Project the data on spanR(u1,...,uk).**
 
-&#10230;
+&#10230; Paso 4: Proyectar los datos en spanR(u1,...,uk).
 
 <br>
 
 **41. This procedure maximizes the variance among all k-dimensional spaces.**
 
-&#10230;
+&#10230; Este procedimiento maximiza la varianza entre todos los espacios k-dimensionales.
 
 <br>
 
 **42. [Data in feature space, Find principal components, Data in principal components space]**
 
-&#10230;
+&#10230; [Datos en el espacio de funciones, Buscar componentes principales, Datos en el espacio de componentes principales]
 
 <br>
 
 **43. Independent component analysis**
 
-&#10230;
+&#10230; Análisis de componentes independientes
 
 <br>
 
 **44. It is a technique meant to find the underlying generating sources.**
 
-&#10230;
+&#10230; Es una técnica destinada a encontrar las fuentes generadoras subyacentes.
 
 <br>
 
 **45. Assumptions ― We assume that our data x has been generated by the n-dimensional source vector s=(s1,...,sn), where si are independent random variables, via a mixing and non-singular matrix A as follows:**
 
-&#10230;
+&#10230; Suposiciones - Suponemos que nuestros datos x han sido generados por el vector fuente n-dimensional s=(s1,...,sn), donde si son variables aleatorias independientes; a través de una matriz A de mezcla y no singular, de la siguiente manera:
+
+&#10230; Suposiciones - Suponemos que nuestros datos x han sido generados por el vector fuente n-dimensional s=(s1,...,sn), donde si son variables aleatorias independientes; a través de una matriz A de mezcla y no singular, de la siguiente manera:
 
 <br>
 
 **46. The goal is to find the unmixing matrix W=A−1.**
 
-&#10230;
+&#10230; El objetivo es encontrar la matriz desmezcladora W=A−1.
 
 <br>
 
 **47. Bell and Sejnowski ICA algorithm ― This algorithm finds the unmixing matrix W by following the steps below:**
 
-&#10230;
+&#10230; Algoritmo ICA de Bell y Sejnowski - Este algoritmo encuentra la matriz desmezcladora W siguiente los siguientes pasos: 
 
 <br>
 
 **48. Write the probability of x=As=W−1s as:**
 
-&#10230;
+&#10230; Escribir la probabilidad de x=As=W−1s como:
 
 <br>
 
 **49. Write the log likelihood given our training data {x(i),i∈[[1,m]]} and by noting g the sigmoid function as:**
 
-&#10230;
+&#10230; Escriba la probabilidad dado nuestros datos de entrenamiento {x(i),i∈[[1,m]]} y denotando g, la función sigmoide, como:
 
 <br>
 
 **50. Therefore, the stochastic gradient ascent learning rule is such that for each training example x(i), we update W as follows:**
 
-&#10230;
+&#10230; Por lo tanto, la regla de aprendizaje de ascenso de gradiente estocástica es tal que para cada ejemplo de entrenamiento x(i), actualizamos W de la siguiente manera:
 
 <br>
 
 **51. The Machine Learning cheatsheets are now available in Spanish.**
 
-&#10230;
+&#10230; Aprendizaje no Supervisado ahora están disponibles en Español.
```

Comment:
"Las hojas de referencia de Machine Learning están ahora disponibles en Español"

It does not say anything about Unsupervised Learning.